### ISSUE TITLE
Command Palette Shows Preview

### ISSUE THREAD

**Issue by exastone** [helix-editor/helix#8665]
Small UX enhancement request.
The default for pickers is to show preview, however this isn't a sensible default for the command palette picker.
Is there a way to distinguish between different pickers to change the default for showing the preview split for command palette picker and file picker?

**Comment by the-mikedavis** [comment]
This should be a small fix. Here: https://github.com/helix-editor/helix/blob/992b7a0c395b5c43e2fd4f6eee38bed8dda9721a/helix-term/src/ui/picker.rs#L798

We should also check that the picker has a preview function:

```rust
let render_preview = self.show_preview && self.file_fn.is_some() && area.width > MIN_AREA_WIDTH_FOR_PREVIEW;
```

**Comment by Utshaan** [comment]
Issue is said to be fixed in version 23.10, and I am facing this issue with that same version

**Comment by the-mikedavis** [comment]
This isn't included in 23.10. 23.10 was cut on 2023-10-24 and https://github.com/helix-editor/helix/pull/8667 was merged on 2023-10-30
